File: 009-triangles/009-triangles.fs
open System.IO


let checkTriangle (n: int list) =
    if n.[0] + n.[1] > n.[2] && n.[1] + n.[2] > n.[0] && n.[0] + n.[2] > n.[1] then 1 
    else 0


let lines = File.ReadLines("C:\Users\wintermute\Documents\Visual Studio 2015\Projects\Codeabbey\Codeabbey\input.txt")
for line in lines do
    line.Split() |> Seq.map int |> Seq.toList |> fun x -> printf "%i " (checkTriangle x)
let x = System.Console.ReadLine()

File: 009-triangles/009-triangles.pl
#!/usr/bin/perl
use strict;
use warnings;

open (IN, "input.txt");
<IN>;
my @output = ();
while (my $input = <IN>) {
    my ($a, $b, $c) = sort {$a <=> $b} split (/\s+/, $input);
    push (@output, ($a + $b >= $c) ? 1 : 0);
}
close (IN);
open (OUT, ">output.txt");
print OUT join (" ", @output);
close (OUT);